### DISCUSSION TITLE
优选IP后大部分IP段都不能用，提示日期问题，但服务端和客户端时间都没问题，根本连不上网络？

### DISCUSSION THREAD

**Discussion by Dell911** [XIU2/CloudflareSpeedTest#323] [讨论]
proxy/http: failed to read response from 91.108.56.102:80 > io: read/write on closed pipe
2022/11/25 21:14:42 [Warning] [2232881172] app/proxyman/inbound: connection ends > proxy/http: connection ends > proxy/http: failed to write response > write tcp 127.0.0.1:10809->127.0.0.1:56231: wsasend: An established connection was aborted by the software in your host machine.

近个把月出现的事情，之前都没事的，现在只有一个网段能用，其他网段都不可以，但是能用的这个网段油管速度慢的一批，毫无体验感，有遇到这个问题的吗？

**Comment by 65654865** [comment]
#250
